receiver/signalfxreceiver/receiver.go

```diff
@@ -253,3 +271,12 @@ func (r *sfxReceiver) failRequest(
 		zap.Error(err), // It handles nil error
 		zap.String("receiver", r.config.Name()))
 }
+
+func jsonResponse(s string) []byte {
+	respBody, err := json.Marshal(s)
+	if err != nil {
+		// This is to be used in initialization so panic here is fine.
```

Comment:
nit: can we call it initJSONResponse or something similar to make the intention clear? Another idea is to use the init() function to make it very explicit. I generally don't like the init func but I guess in this case is acceptable. Your call.

Reply:
I like rename but since is for private vars on the package I won't add a init().